docs/source/developer_guide/guides/9_control_messages.md

```diff
@@ -88,12 +88,17 @@ data = cudf.DataFrame()
 msg_meta = MessageMeta(data)
```

Comment:
Update the docs above to explicitly mention that in addition to the deprecation in 24.06, in 24.10 MultiMessage was removed.